### PR TITLE
Update Gem version, update specs.

### PR DESCRIPTION
**Pull request by eschlange**
* Update TFD Engine gem to latest.
* Update specs to look for new prompt value and page text.

[#94335536]

### CODE REVIEW

File: spec/features/activity_sharing_spec.rb [hunk 1]
```diff
@@ -37,7 +37,7 @@
 
     it "shows social participants option to share" do
       expect(page).to have_text "We want you to plan one fun thing"
-      expect(page).to have_text "Share the content of this activity?"
+      expect(page).to have_text "Share the content of this planned_activity?"
```
Comment (ericcf) [inline spec/features/activity_sharing_spec.rb]: Do you think the view should remove the underscore before rendering this?

Comment (eschlange) [inline spec/features/activity_sharing_spec.rb]: Hah, yes. Didn't catch that, I think I'll need to update the engine to fix this...

**Comment by eschlange** [comment]
@ericcf I updated the display logic to remove underscores and updated the spec.

**Comment by ericcf** [comment]
Thanks.

**Pull request merged**